src/main/java/com/cburch/draw/tools/DrawingAttributeSet.java

```diff
@@ -19,13 +19,13 @@
 import com.cburch.logisim.data.AttributeSet;
 import com.cburch.logisim.util.EventSourceWeakSupport;
 import com.cburch.logisim.util.UnmodifiableList;
+
```

Comment:
What is that for? It adds no value and I think separating local imports from other is pointless - your IDE will hide them all anyway so no point for having diffs like this.

Reply:
Personally, I don't have a problem with this added line. IMHO, it's a matter of taste.

Reply:
I mostly mean that it rather matter of IDE settings. I also do not really care that much. Just asked what for as I may miss something still.